DXRModules/DeusEx/Classes/DXREnemies.uc

```diff
@@ -94,6 +95,7 @@ function CheckConfig()
     AddRandomEnemyType(class'#var(prefix)MilitaryBot', 2, MJ12);
     AddRandomEnemyType(class'#var(prefix)SpiderBot2', 2, MJ12);//little spider
     AddRandomEnemyType(class'#var(prefix)SpiderBot', 2, MJ12);//big spider
+    AddRandomEnemyType(class'#var(prefix)GrayBaby', 2, MJ12);
```

Comment:
I feel like 2 is a bit high? same number of babies as regulars? is MJ12 really that evil?

Reply:
I'm fine with it going down - the odds are still pretty low and they're pretty weak though